frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -34,32 +33,32 @@ import {
     type IdimProxyIdirInfo,
 } from 'fam-api';
 
+import { IconSize } from '@/enum/IconEnum';
+
 const defaultFormData = {
     domain: domainOptions.IDIR,
     userId: '',
-    forestClientNumber: '',
+    forestClientNumberList: [],
+    forestNumberError: false,
     role_id: null as number | null,
 };
 const formData = ref(JSON.parse(JSON.stringify(defaultFormData))); // clone default input
+
+const forestClientInactive = ref<boolean>(false);
+const forestClientDuplicate = ref<boolean>(false);
```

Comment:
as another comment, we might need another variable here "forestClientInvalid", in the case the forest client number is not found

Reply:
Good idea, that makes sense.